### PR TITLE
implemented exercise isogram

### PR DESCRIPTION
**Pull request by querenker**

### CODE REVIEW

File: exercises/isogram/src/isogram.clj [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(ns isogram
```
Comment (yurrriq) [inline exercises/isogram/src/isogram.clj]: This file needs to be renamed to `example.clj`.

File: config.json [hunk 2]
```diff
@@ -52,7 +52,8 @@
     "minesweeper",
     "change",
     "pangram",
-    "wordy"
+    "wordy",
+	"isogram"
```
Comment (yurrriq) [inline config.json]: The indentation of `"isogram"` needs to be adjusted to match that of the rest of the `"problems"`.

Comment (querenker) [inline config.json]: @yurrriq Thank you for your annotations. Now it should work and look properly :)

**Comment by yurrriq** [comment]
Thanks! LGTM.

**Pull request merged**